dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -107,8 +107,14 @@ public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String d
                 Map.Entry<String, String> doiEntry = doiMap.entrySet().iterator().next();
                 String doi = doiEntry.getValue();
 
-                createAliasAndsetUpRelatedIdentifiers(depositMetadata, dockstoreGA4GHBaseUrl, dockstoreUrl,
-                        workflowUrl, workflow, workflowVersion, doi);
+                // Create the alias and make sure the alias is valid
+                // If it is not acceptable then an exception is generated
+                // in which case a deposition resource may be left on Zenodo
+                // that the user will have to clean up manually
+                doiAlias = createAliasUsingDoi(doi);
+
+                setMetadataRelatedIdentifiers(depositMetadata, workflowResource.getDockstoreGA4GHBaseUrl(),
```

Comment:
Rather than pass around the workflowResource in order to get access to a couple static strings, they can simply be accessed statically or passed as parameters.

Reply:
fixed, now pass only strings